tests/test_continuous.py

```diff
@@ -146,7 +146,7 @@ def test_ddpg_eval_env():
     model = DDPG("MlpPolicy", "Pendulum-v0", nb_rollout_steps=5,
                 nb_train_steps=2, nb_eval_steps=10,
                 eval_env=eval_env, verbose=0)
-    model.learn(1000)
+    model.learn(300)
```

Comment:
Use `NUM_TIMESTEPS`? Same below

Reply:
probably, I need to see as all algorithms don't have the same default...